woocommerce/woocommerce
# Order Item Data Store

This PR implements data stores for all order item types. There is an abstract base class for order item stores, and then smaller stores that handle each specific set of data. This also updates the main data classes to use the new set_prop and get_prop methods.

#### Testing:

Unit Tests: All tests pass. `order/crud` tests have some line item tests specifically.

Cart/Ordering:
* Add a few different product types to your cart - enable things like shipping fees or taxes.
* Checkout and verify that info shows correctly on the order summary page (product name, cost, tax info, etc).

Admin:
* Verify that info shows correctly for the order line items in wp-admin.
* Add new fees, shipping costs, or products to the order. Save and verify everything works and displays correctly.
